swap_meet/clothing.py

```diff
@@ -1,2 +1,12 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, condition = "", age = 0):
+        self.category = "Clothing"
+        self.condition = condition
+        self.age = age
```

Comment:
This works but I think we can make this more efficient!

Given that we repeat L6-8 in the parent class `Item`'s initializer, we can simplify this bit to:
```
super().__init__(category="Clothing", condition=condition, age=age)
```